### PR TITLE
Add text field for time votes: version with public usernames

### PR DESCRIPTION
**Pull request by alice-blue**
This is a draft. I made a start at a version that shows names publicly, but it has some problems.

### CODE REVIEW

**Comment by dfabulich** [comment]
I'm going to do some copying and pasting and make a separate PR with just public user names after a cutoff date.

File: www/viewgame [hunk 1]
```diff
@@ -1454,14 +1459,17 @@ function updateUnwishList(id, stat)
...
+<li>Vote on how long it takes to get to one final ending without relying extensively on a walkthrough.</li>
 <li>Time spent away from the game doesn't count.</li>
+<li>To help players gauge how long a game with puzzles might take, you can note if your time is with or without hints.</li>
 <li>Times don't need to be exact. Good-faith estimates are fine.</li>
 </ul></details>
 <p><label for='hours'>Hours:</label>
 <input type='number' name='hours' id='hours' min=0 max=200>
 <label for='minutes'>Minutes:</label>
 <input type='number' name='minutes' id='minutes' min=0 max=59></p>
+<p><label for='timeNote'>Note:</label>
```
Comment (dfabulich) [inline www/viewgame:1471]: Maybe call it a "Time Note"?

File: www/viewgame [hunk 2]
```diff
@@ -1454,14 +1459,17 @@ function updateUnwishList(id, stat)
...
+<input type='text' name='timeNote' id='timeNote' maxlength='200' size='26'></p>
```
Comment (dfabulich) [inline www/viewgame:1472]: Maybe add a [`placeholder`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/placeholder) attribute, like: `placeholder='e.g. "Completionist" or "No hints"'`

**Comment by alice-blue** [comment]
Working on resolving this

**Comment by alice-blue** [comment]
@dfabulich I tried to resolve it, but I can't get it to display the note. Maybe you can tell what I'm doing wrong.

**Comment by alice-blue** [comment]
@dfabulich I think this is working now. Let me know if you want me to copy and paste everything into a new PR so it's not as messy.

**Pull request closed without merging**